ui/js/controllers/bugfiler.js

```diff
@@ -96,6 +101,11 @@ treeherder.controller('BugFilerCtrl', [
                 }
             }
 
+            // We don't want to include "REFTEST" when it's an unexpected pass
+            if (summary[0].search("REFTEST TEST-UNEXPECTED-PASS") >= 0) {
```

Comment:
It would be great to factor this out to avoid the duplication (the list of things applied in both places is probably going to grow in the future too).